packages/kit/package.json

```diff
@@ -4,6 +4,8 @@
   "repository": "nuxt/nuxt",
   "license": "MIT",
   "type": "module",
+  "main": "./dist/index.mjs",
+  "module": "./dist/index.mjs",
```

Comment:
Module field is only picked by bundlers are you sure we need it?

Reply:
We might get away with just main 👍

Reply:
Oh dear. It's worse than that. We actually need a `require` export condition.